File: class16/cpp/AP1601-Triangulo.cpp
#include <iostream>
int main() {
    double a, b, c;
    std::cout << "Lado a: "; std::cin >> a;
    std::cout << "Lado b: "; std::cin >> b;
    std::cout << "Lado c: "; std::cin >> c;
    if (a < b + c && b < a + c && c < a + b) {
        if (a == b && b == c) std::cout << "Triangulo EQUILATERO" << std::endl;
        else if (a == b || b == c || a == c) std::cout << "Triangulo ISOSCELES" << std::endl;
        else std::cout << "Triangulo ESCALENO" << std::endl;
    } else {
        std::cout << "Nao formam um triangulo" << std::endl;
    }
    return 0;
}

File: class16/java/AP1601-Triangulo.java
import java.util.Scanner;

public class Triangulo {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Lado a: ");
        double a = sc.nextDouble();
        System.out.print("Lado b: ");
        double b = sc.nextDouble();
        System.out.print("Lado c: ");
        double c = sc.nextDouble();
        if (a < b + c && b < a + c && c < a + b) {
            if (a == b && b == c) System.out.println("Triangulo EQUILATERO");
            else if (a == b || b == c || a == c) System.out.println("Triangulo ISOSCELES");
            else System.out.println("Triangulo ESCALENO");
        } else {
            System.out.println("Nao formam um triangulo");
        }
    }
}
